pkg/get/tools.go

```diff
@@ -1945,5 +1945,34 @@ https://github.com/{{.Owner}}/{{.Repo}}/releases/download/{{.Version}}/{{.Name}}
 			{{ .Name }}_{{ .VersionNumber }}_{{ $osStr }}_{{ $archStr }}{{ $extStr }}
 			`,
 		})
+
+	tools = append(tools,
+		Tool{
+			Owner:       "sunny0826",
+			Repo:        "kubecm",
+			Name:        "kubecm",
+			Version:     "0.16.2",
```

Comment:
I don't think using `Version` here is necessarry, I assume that GitHub can determine the version.

edit: It can and works perfectly fine.

Reply:
> I don't think using `Version` here is necessarry, I assume that GitHub can determine the version.
> 
> edit: It can and works perfectly fine.


modified in new commit [378e192](https://github.com/karthick-kk/arkade/commit/378e19209bbd68a3dd4246ed90a67d4b4b82af40)